### ISSUE TITLE
ShadSlider doesn't have a focus indicator

### ISSUE THREAD

**Issue by helightdev** [nank1ro/flutter-shadcn-ui#204]
### Steps to reproduce

Select a slider using Tab

### Expected results

The official shadcn implementation draws a second circle arround the slider when selected.

### Actual results

When selecting a slider using Tab, no indication is given that the slider is selected even though it can be manipulated.

### shadcn_ui version

main

### Platform

Web

### Code sample

-

### Screenshots or Video
What it should look like:
![Screenshot 2024-11-29 221842](https://github.com/user-attachments/assets/7f3defff-66d9-4041-aeef-881fefae44a9)

### Flutter Doctor output

.

**Comment by helightdev** [comment]
I changed the thumb and slider state to draw an indicator like in the official shadcn implementation.

**Comment by nank1ro** [comment]
Hey @helightdev can you address the required changes in the PRs?
Otherwise I will continue them in the next days

**Comment by helightdev** [comment]
Sorry, I'm currently not in my development environment, it would probably be faster to just make the changes yourself.

**Comment by nank1ro** [comment]
done in 5bf80f2fab002d0acc82e5d4b758e7f84e65982f

a new version will be released today
